bcipy/simulator/helpers/sim_viz.py

```diff
@@ -0,0 +1,68 @@
+import logging as logger
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+from bcipy.helpers.parameters import Parameters
+
+logger.getLogger().setLevel(logger.INFO)
+
+
+def plot_comparison_records(records, outdir, title="response_values", y_scale="log"):
```

Comment:
Be sure to add type annotations to all of your function signatures, as well as a docstring describing what the function does and what the parameters are used for.